frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/validations/vm-settings-tab-validation.ts

```diff
@@ -144,6 +145,14 @@ const asVMSettingsFieldValidator = (
     subject: getFieldTitle(iGetFieldKey(field)),
   });
 
+const validateSource = (value: string): ValidationObject => {
```

Comment:
it should all be validated here. This is the correct place in the wizard lifecycle where to set it. All logic depends on this behaviour.

Reply:
logic moved to validation